src/test/java/org/apache/commons/text/lookup/StringLookupFactoryTest.java

```diff
@@ -257,6 +257,16 @@ void testSingletons() {
         assertSame(XmlEncoderStringLookup.INSTANCE, stringLookupFactory.xmlEncoderStringLookup());
     }
 
+    /**
+     * Tests that we return the singleton.
+     */
+    @Deprecated
```

Comment:
We don't use the `Deprecated` annotations on tests.

Reply:
Ok, but it is used in some places already in some test-classes.